src/picongpu/include/plugins/PluginController.hpp

```diff
@@ -114,25 +105,19 @@ class PluginController : public ILightweightPlugin
     typedef Visualisation<PIC_Electrons, PngCreator> ElectronsPngBuilder;
     typedef PngPlugin<ElectronsPngBuilder > PngImageElectrons;
 #endif
-    typedef ParticleDensity<PIC_Electrons, DensityToBinary, float_X> ElectronsBinaryDensityBuilder;
 
 #if(ENABLE_HDF5 == 1)
     /* speciesParticleShape::ParticleShape::ChargeAssignment */
     typedef PhaseSpaceMulti<particles::shapes::Counter::ChargeAssignment, PIC_Electrons> PhaseSpaceElectrons;
 #endif
 #if(SIMDIM==DIM3)
-#if(PIC_ENABLE_PNG==1)
-    typedef heiko::ParticleDensity<PIC_Electrons> HeikoParticleDensity;
```

Comment:
@f-schmitt-zih we just removed the `heiko::` namespace. can you believe that? ;)